textattack/shared/scripts/run_attack_args_helper.py

```diff
@@ -335,3 +349,24 @@ def parse_logger_from_args(args):# Create logger
     if not args.disable_stdout:
         attack_log_manager.enable_stdout()
     return attack_log_manager
+
+def parse_checkpoint_from_args(args):
+    if args.checkpoint_resume:
+        if args.checkpoint_resume.lower() == 'latest':
+            chkpt_files = [f for f in os.listdir(args.checkpoint_dir) if f.endswith('.ta.chkpt')]
+            latest_file = max(chkpt_files)
```

Comment:
are you sure this works how you intend it to? what about this:

```
chkpt_files = ['check_9.chkpt', 'check_10.chkpt']
max(chkpt_files)
```

produce `check_9` instead of `check_10`?

Reply:
Default format is `{time}.ta.chkpt`, so `max` should select the latest item. The way it is set up right now is that users cannot set the name of checkpoint files.